packages/storage/src/database/global/metadata-database.ts

```diff
@@ -114,10 +114,14 @@ export class ModelMetadataDatabase extends ModelBasic<
         appPublicKey: string;
       */
       await collection.index({ databaseName: 1 }, { unique: true, session });
-      await collection.index({ databaseOwner: 1 }, { unique: false, session });
       await collection.index({ merkleHeight: 1 }, { unique: false, session });
       await collection.index({ appPublicKey: 1 }, { unique: false, session });
-
+      // Compound index
+      await collection.index(
+        { databaseName: 1, databaseOwner: 1, mekrleHeight: 1 },
```

Comment:
typo `mekrleHeight`

also this unique index allows a database owner to have multiple databases with the same `databaseName` (but different `merkleHeight`)

shouldn't we create a unique index on (databaseOwner, databaseName) and a normal index on (databaseOwner, merkleHeight) instead?

Reply:
solved